src/main/java/com/es/phoneshop/web/ProductDemodataServletContextListener.java

```diff
@@ -0,0 +1,113 @@
+package com.es.phoneshop.web;
+
+import com.es.phoneshop.dao.ProductDao;
+import com.es.phoneshop.dao.impl.ArrayListProductDao;
+import com.es.phoneshop.model.Product;
+
+import javax.servlet.ServletContextEvent;
+import javax.servlet.ServletContextListener;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.Month;
+import java.util.*;
+
+public class ProductDemodataServletContextListener implements ServletContextListener {
+
+    @Override
+    public void contextInitialized(ServletContextEvent servletContextEvent) {
+        String demodata = servletContextEvent.getServletContext().getInitParameter("demodata");
+
+        if (demodata.toLowerCase().equals("enable")) {
```

Comment:
str.equalsIgnoreCase("enable") is same as str.toLowerCase().equals("enable"), but one method